test/instrumentation/sidekiq-client_test.rb

```diff
@@ -35,6 +39,36 @@ def test_enqueue
     assert_normal_trace_recorded(job)
   end
 
+  def test_enqueue_as_root_exit_span
+    clear_all!
+    ::Instana.config[:allow_exit_as_root] = true
+    disable_redis_instrumentation
+    ::Sidekiq::Client.push(
+      'queue' => 'some_random_queue',
+      'class' => ::SidekiqJobOne,
+      'args' => [1, 2, 3],
+      'retry' => false
+    )
+    ::Instana.config[:allow_exit_as_root] = false
+    enable_redis_instrumentation
+
+    queue = ::Sidekiq::Queue.new('some_random_queue')
+    job = queue.first
+
+    assert_job_enqueued(job)
+    spans = ::Instana.processor.queued_spans
+    assert_equal 1, spans.length
+
+    first_span = spans[0]
+
+    assert_equal :'sidekiq-client', first_span[:n]
+    assert_equal 'some_random_queue', first_span[:data][:'sidekiq-client'][:queue]
+    assert_equal 'SidekiqJobOne', first_span[:data][:'sidekiq-client'][:job]
+    assert_equal "false", first_span[:data][:'sidekiq-client'][:retry]
+    assert       first_span[:data][:'sidekiq-client'][:'redis-url']
```

Comment:
Just a small formatting suggestion. Looks good to me otherwise.
```suggestion
    assert  first_span[:data][:'sidekiq-client'][:'redis-url']
```

Reply:
Fixed.